src/dnddiceroller/DiceTower.java

```diff
@@ -4,25 +4,36 @@
 import java.util.List;
 
 /**
- * Dice Tower.
- * A Dice Tower is a tool used by serious gamers use to roll many dice at once.
- * It looks like this https://www.miniaturescenery.com/Images/PortableDiceTowerLarge.jpg
- * An instance of a dice tower is defined by the number panels it contains to help
- * provide a more regular distribution of die values. The die bounce from panel to
- * panel until they exit the dice tower at the bottom tray.
- * A dice tower will accept a collection of dice and reports their results when
- * they reach the tray at the bottom
- * @author Paul Scarrone
+ * @author Steven Plas
  */
 public class DiceTower {
   final int PANEL_COUNT = 3;
   List<Die> dice;
+  private int dieSum;
 
   public DiceTower() {
 	this.dice = new ArrayList();
+        dieSum = 0;
   }
   
-  public DiceTower(List dice) {
+  public DiceTower(List dice){
 	this.dice = dice;
+        dieSum = 0;
   }
+  
+  public int getDieSum(){
+      return dieSum;
+  }
+  
+  public void dropDice() {
+       int i = 0;
+       for (Die die : dice){
```

Comment:
But this is a double loop.

For each die on :30 you roll all the dice again meaning their values keep changing